### PR TITLE
Rename 'Interface' to 'ModelType'

### PR DESCRIPTION
**Pull request by object88**
As it says on the tin.

### CODE REVIEW

**Comment by pbomb** [comment]
Unfortunately, I just pushed some conflicts. Could you resolve those? Then I can merge this.

**Comment by object88** [comment]
My pleasure.

**Comment by object88** [comment]
Looks like it should be good to go now.

File: src/helpers/withoutModelTypeSuffix.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// @flow
+
+const intfLength = 'ModelType'.length;
```
Comment (pbomb) [inline src/helpers/withoutModelTypeSuffix.js]: Can you rename this variable?

Comment (object88) [inline src/helpers/withoutModelTypeSuffix.js]: Yep.

**Comment by object88** [comment]
I noticed a couple of other places that I accidentally left "interface"; fixing.

**Comment by object88** [comment]
Fixed `intfName` and instance of "Interface" in the README.md.

**Pull request merged**